pkg/apis/kops/cluster.go

```diff
@@ -260,6 +260,12 @@ const (
 type LoadBalancerAccessSpec struct {
 	Type               LoadBalancerType `json:"type,omitempty"`
 	IdleTimeoutSeconds *int64           `json:"idleTimeoutSeconds,omitempty"`
+	// AdditionalSecurityGroups attaches additional security groups (e.g. sg-123456)
+	// This API is currently under development.
+	AdditionalSecurityGroups []string `json:"additionalSecurityGroups,omitempty"`
+	// SecurityGroup is the id of the shared security group to use for the InstanceGroupSpec
+	// This API is currently under development.
+	SecurityGroup *string `json:"securityGroup,omitempty"`
```

Comment:
Looks good, but I think the naming or comment could potentially be reworded so it's clearer. Maybe (if I have understood this correctly):
```
// SecurityGroupID is the ID of a pre-existing security group to use on the resource. This ID
// will be referenced in rules within the InstanceGroupSpec Security Groups.
// This API is currently under development.
```

Reply:
Wordsmithing would be awesome